test/lambdaInstructionTest.cpp

```diff
@@ -41,7 +41,6 @@
 #include "data/untypedSharedPtr.h"
 #include "instructions/addPrimitiveType.h"
 #include "instructions/lambdaInstruction.h"
-#include "instructions/multByConstParam.h"
 #include "instructions/set.h"
 
```

Comment:
Did you test LambdaInstructions with Constant? (If it can work.. a unit test would be great to verify it)